js/templates/modals/editListing/variant.html

```diff
@@ -19,18 +19,7 @@
     if (variantsErrs.length) print(ob.formErrorTmpl({ errors: variantsErrs }));
   %>
   <div class="flexRow marginTopAuto">
-    <div class="select2TagFlexWrap">
-      <div class="select2TagsWrap">
-        <!-- WARNING! Styling is dependant on the following elements being in this order. -->
-        <select multiple name="variants" class="clrBr clrP select2Tags" style="width: 100%">
-          <% ob.variants.forEach(variant =>
-              print(`<option value="${variant._clientID}<===>${variant.name}" selected>${variant.name}</option>`)) %>
-        </select>
-        <div class="placeholder tagsPlaceholder js-variantChoicesPlaceholder"><%= ob.polyT('editListing.variants.choicesPlaceholder') %></div>
-        <!-- needed to properly hide the select2 dropdown which we don't want on a tagging field -->
-        <div class="tagsDropdown js-dropDownContainer"></div>
-      </div>
-    </div>
+    <select multiple name="variants" class="clrBr clrP clrSh2 hideDropDown flexExpand" placeholder="<%= ob.polyT('editListing.variants.choicesPlaceholder') %>"></select>
```

Comment:
Note the `hideDropDown` class. One of the quirks with selectize is if you delete an existing tag and then start typing it in again, it will appear in a drop down of available options. On most of the tagging fields we don't want such behavior - we essentially want a text field that contains tags. Use the `hideDropDown` class on these fields.

It will style the drop down to be hidden. It is also used by the hacked selectize lib to prevent certain related behavior (e.g. an enter press auto adding an option from the hidden menu).